src/Presentation/Nop.Web/Views/Product/_ProductSpecifications.cshtml

```diff
@@ -6,21 +6,21 @@
         <div class="title">
             <strong>@T("Products.Specs")</strong>
         </div>
-        <div class="table-wrapper">
-            <table class="data-table">
-                <colgroup>
-                    <col width="25%" />
-                    <col />
+        <div role="table" class="table-wrapper">
```

Comment:
Could you please clarify why role="table" is added here if below we see the actual table?

Reply:
Hi yes that is the way the table role works.  Please review the documentation outlined here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role

Reply:
But we use a native <table> HTML element as described in this article: "Using a native HTML table element whenever possible is strongly encouraged."
So why do you use such roles as table, row, cell etc. if this is a native table element?

Reply:
According to ADA guidelines, using the role attributes for table, row and cell is necessary for screen readers.  I am simply following the guidelines that are required for ADA compliance. I do not know the reasoning behind that I am told to do, but I don't see how it will cause any issues to include it regardless.

Reply:
I see why now. It's because we use this table tag for layout purposes, not to represent a real table data. Ok, let's leave it this way.